hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.test.tsx

```diff
@@ -80,7 +91,12 @@ describe("team pending order table", () => {
         });
     });
     test("Check all button checks and unchecks every row", async () => {
-        const { getByTestId } = render(<TeamPendingOrderTable />);
+        const { getByTestId } = render(
+            <TeamPendingOrderTable
+                orders={mockPendingOrdersInTable}
+                optionalHardware={mockHardwareWithBuffer}
+            />
+        );
```

Comment:
could you add a test that tests that the thunk is dispatched.
The main thing you need to test is that the state changes correctly and the table disappears (for a cancelled order) or the status pill changes (for a ready-for-pickup order)

Reply:
how should I wait for the thunk to complete before testing to see if these changes appear?